project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module

```diff
@@ -117,7 +116,13 @@ function c4m_features_homepage_c4m_featured_groups_get_groups() {
  * Loads list of nodes, sorts them and renders them as block list.
  */
 function c4m_features_homepage_alphabetically_render_node_list($node_ids) {
+  module_load_include('inc', 'c4m_features_homepage', 'plugins/content_types/my_groups/my_groups');
   $items = node_load_multiple(array_keys($node_ids));
+
+  // Removes groups where the logged in user has no access.
+  $items = array_filter($items, 'c4m_features_homepage_node_access_view_filter');
+  $items = array_slice($items, 0, 3);
```

Comment:
@DavidHernandez 
Please move this magic number to a constant (you can use `[ci skip]` on the commit)